rest-service/manager_rest/storage/resource_models.py

```diff
@@ -420,6 +437,14 @@ def _list_workflows(deployment_workflows):
                          parameters=wf.get('parameters', dict()))
                 for wf_name, wf in deployment_workflows.items()]
 
+    @property
+    def latest_execution_status(self):
+        if not self.latest_execution:
+            return None
+        return DeploymentState.EXECUTION_STATES_SUMMARY.get(
```

Comment:
actually, there is a reason which is if you check the deployment statuses spec you will notice there is a translation needed here since on the UI there are not using the all the latest execution statuses and they should be mapped like this 
```
Last workflow state

Completed

Failed

In progress – force cancelling, cancelling, kill cancelling, started, pending

Cancelled
```

Reply:
so basically we have 4 latest execution statuses we are interested in